### PR TITLE
CLOUDP-231603: Fix updating dependabot licenses

### PR DESCRIPTION
**Pull request by josvazg**
✅ Managed to fix dependabot PR #1408 (launched manually)

### All Submissions:

* [X] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

**Comment by josvazg** [comment]
I manually forced a `dependabot` re-check following instruction from https://github.com/dependabot/dependabot-core/issues/2980

Go to https://github.com/mongodb/mongodb-atlas-kubernetes/network/updates
Open `Recent update jobs` for `go mod` and click `Check for updates`

File: .github/workflows/update-licenses.yml [hunk 1]
```diff
@@ -12,22 +16,31 @@ jobs:
...
-            dependabot skip
-          branch: ${{ github.head_ref }}
+      - name: Commit as needed
+        run: |
+          if [[ $(git diff --stat) != '' ]]; then
+            echo 'Committing changes'
+            git config user.email "akobot@ako-team-fake.mongodb.com"
+            git config user.name "AKOBot"
+            git add .
+            git commit -m "Fix licenses after dependabot changes"
```
Comment (roothorp) [inline .github/workflows/update-licenses.yml:42]: I think [including dependabot skip](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates#allowing-dependabot-to-rebase-and-force-push-over-extra-commits) in the commit message means you avoid the error you're getting.

**Review by helderjs — APPROVED** [review]

**Review by igor-karpukhin — APPROVED** [review]

File: .github/workflows/update-licenses.yml [hunk 2]
```diff
@@ -12,22 +16,31 @@ jobs:
...
-          commit_message: |
-            Updated dependency licenses
-
-            dependabot skip
-          branch: ${{ github.head_ref }}
+      - name: Commit as needed
+        run: |
+          if [[ $(git diff --stat) != '' ]]; then
+            echo 'Committing changes'
+            git config user.email "akobot@ako-team-fake.mongodb.com"
```
Comment (igor-karpukhin) [inline .github/workflows/update-licenses.yml:39]: Is this parameter mandatory?

Comment (josvazg) [reply .github/workflows/update-licenses.yml:39]: Yes, it seems git complains otherwise. Also it is best we know it was the bot that did the PR.

**Review by roothorp — APPROVED** [review]

**Pull request merged**